src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -696,8 +696,129 @@ private Task WriteSuffix()
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private unsafe void EmitHttp11(ref MemoryPoolIterator end)
+        {
+            byte* b;
+            
+            if (end.GetRawBuffer(9, out b))
+            {
+                 // Emit directly
+                *(long*)b = 0x312E312F50545448;
```

Comment:
@halter73 I mean in the sense that depending on the processor architecture, the instruction could fail or result in a silent invalid memory access.

https://www.kernel.org/doc/Documentation/unaligned-memory-access.txt

> Why unaligned access is bad
> 
> The effects of performing an unaligned memory access vary from architecture
> to architecture. It would be easy to write a whole document on the differences
> here; a summary of the common scenarios is presented below:
> 
>  - Some architectures are able to perform unaligned memory accesses
>    transparently, but there is usually a significant performance cost.
>  - Some architectures raise processor exceptions when unaligned accesses
>    happen. The exception handler is able to correct the unaligned access,
>    at significant cost to performance.
>  - Some architectures raise processor exceptions when unaligned accesses
>    happen, but the exceptions do not contain enough information for the
>    unaligned access to be corrected.
>  - Some architectures are not capable of unaligned memory access, but will
>    silently perform a different memory access to the one that was requested,
>    resulting in a subtle code bug that is hard to detect!